www/app/search/search.controller.spec.js

```diff
@@ -0,0 +1,55 @@
+/* jshint -W117, -W030 */
+describe('SearchController', function() {
+  var controller,
+      deferredSearch,
+      deferredToTablePage,
+      tablesServiceMock,
+      yelpServiceMock;
+
+  // Load the App Module to access controllers
+  beforeEach(module('app'));
+
```

Comment:
beforeEach takes a function as an argument, and then you can inject inside that function as well as load the App.